src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -76,14 +76,33 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
             if (File.Exists(documentationPath))
                 return true;
 
+            //try with uppercase XML for linux
+            documentationPath = PathHelper.replaceExtension(documentationPath, ".XML");
```

Comment:
@ramramps something like File.Exists([documentationPath 1, documentationPath 2]) => path that exists?